cmake/module/GenerateMPCode.cmake

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2023-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+function(target_capnp_sources target)
+  foreach(capnp_file IN LISTS ARGN)
+    message("      ++++++++++++++++++ ${capnp_file}")
```

Comment:
Should this line be removed? Guessing this was for debug

Reply:
oops...

Reply:
Removed.